rallyrolebot/cogs/rally_cog.py

```diff
@@ -32,3 +34,45 @@ async def cog_command_error(self, ctx, error):
     @commands.dm_only()
     async def set_rally_id(self, ctx, rally_id):
         data.add_discord_rally_mapping(ctx.author.id, rally_id)
+
+    @commands.command(name="price", help="Get the price data of a coin")
+    @validation.is_valid_coin()
+    async def price(self, ctx, coin_symbol):
```

Comment:
```suggestion
        async def price(self, ctx, coin_symbol : converters.CreatorCoinConverterGoesHere ):
```
This would be better done by creating discord.py [Advanced Converter](https://discordpy.readthedocs.io/en/latest/ext/commands/commands.html#converters) named ```CreatorCoin```  under utils.converters. 

We typically do not validate arguments via decorators (that can get quite messy).